### ISSUE TITLE
Accessibility: Textual alternatives for unicode symbols missing

### ISSUE THREAD

**Issue by nkreer** [kellerben/dudle#129]
The symbols used in the poll table do not have a suitable textual label for e.g. screen reader users. This could be done via aria-label or a an accessible popover containing a localized string like "yes", "no" and "maybe" as used in the code.   

I'd appreciate a fix very much as this seems like a particularly easy thing to fix in poll.rb starting in line 285.

**Comment by kellerben** [comment]
you are right, this would be an easy fix. Wouldn't it be better to add this to the title in poll.rb line 290?

**Comment by kellerben** [comment]
I added the aria-label to the input. In addition, I added the unicode symbol to the title.
Can you please review it here:
https://dudle.inf.tu-dresden.de/coffee/
the code is here:
https://github.com/kellerben/dudle/blob/master/poll.rb#L290-L291

**Comment by bkmgit** [comment]
https://wave.webaim.org indicates the labels appear, though adding language used for the page would likely also be helpful for screen readers.
